### ISSUE TITLE
hydrating client stores from server + proxies

### ISSUE THREAD

**Issue by babsonmatt** [mschipperheyn/normalizr-immutable#8]
Any ideas on how to use proxies if the stores come over prepopulated from the server?  Seems like you'd have to de-normalize and normalizr-immutable?

**Comment by mschipperheyn** [comment]
Yeah, that would sense to me. I don't have this use case myself. Since normalization and de-normalization is a one command thing, this sounds like a good idea.

**Comment by mschipperheyn** [comment]
Also, when you execute toJSON(), it will retrieve the proxied object and not the proxy itself
